entity-framework/core/miscellaneous/testing/testing-sample.md

```diff
@@ -95,29 +97,29 @@ Most validation and error handling have been removed to reduce clutter.
 ## The Tests
 
 The tests are organized to run with multiple database provider configurations:
-* The SQL Server provider, which is the provider used by the application
-* The SQLite provider
-* The SQLite provider using in-memory SQLite databases
-* The EF in-memory database provider
+
+- The SQL Server provider, which is the provider used by the application
+- The SQLite provider
+- The SQLite provider using in-memory SQLite databases
+- The EF in-memory database provider
 
 This is achieved by putting all the tests in a base class, then inheriting from this to test with each provider.
 
 > [!TIP]
 > You will need to change the SQL Server connection string if you're not using LocalDB.
-
-> [!TIP]
-> See [Testing with SQLite](xref:core/miscellaneous/testing/sqlite) for guidance on using SQLite for in-memory testing. 
+> See [Testing with SQLite](xref:core/miscellaneous/testing/sqlite) for guidance on using SQLite for in-memory testing.
```

Comment:
Not sure if it's a good idea to combine the tips which are unrelated... If this is just about removing a warning, maybe better to suppress the warning here instead somehow?

Reply:
I don't think they are unrelated. Or else the sqlite testing should go to a different place.